js/examples/notebooks/langchain/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "dependencies": {
+        "@arizeai/openinference-instrumentation-langchain": "^0.2.0",
+        "@arizeai/openinference-semantic-conventions": "^0.10.0",
+        "@langchain/core": "^0.2.30",
+        "@langchain/openai": "^0.2.0",
+        "@opentelemetry/api": "^1.9.0",
+        "@opentelemetry/exporter-trace-otlp-proto": "^0.50.0",
+        "@opentelemetry/resources": "^1.25.1",
+        "@opentelemetry/sdk-trace-base": "^1.25.1",
+        "@opentelemetry/sdk-trace-node": "^1.25.1",
+        "getpass": "^0.1.7",
+        "prompt-sync": "^4.2.0"
+    }
+}
```

Comment:
The rest of the package is not needed?

Reply:
doesn't seem like it, it works just fine with just the deps